platform-ui-skin/src/main/webapp/skin/bootstrap/less/variables.less

```diff
@@ -173,7 +173,6 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 // Sprite icons path
 // -------------------------
-@iconSpritePath:          "../img/glyphicons-halflings.png";
 @iconWhiteSpritePath:     "../img/glyphicons-halflings-white.png";
```

Comment:
to do same thing for this property IMO and its usage since it's not correctly referenced neither.

Reply:
done